### PR TITLE
feat(pie-tag): DSW-1519 initial component shell

### PR DESCRIPTION
**Pull request by dandel10n**
---
"@justeattakeaway/pie-tag": minor
---

[Added] - pie-tag component shell

---
"pie-monorepo": minor
---

[Added] - pie-tag project label

---
"pie-storybook": minor
---

[Added] - pie-tag stories

### CODE REVIEW

File: packages/components/pie-tag/package.json [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    "test:visual": "run -T cross-env-shell PERCY_TOKEN=${PERCY_TOKEN_PIE_TAG} percy exec --allowed-hostname cloudfront.net -- npx playwright test -c ./playwright-lit-visual.config.ts",
+    "test:visual:ci": "yarn test:visual"
+  },
+  "author": "Just Eat Takeaway.com - Design System Team",
+  "license": "Apache-2.0",
+  "devDependencies": {
+    "@justeattakeaway/pie-components-config": "workspace:*"
+  },
+  "dependencies": {
+    "@justeattakeaway/pie-webc-core": "workspace:*"
```
Comment (xander-marjoram) [inline packages/components/pie-tag/package.json:34]: This is fine for the shell PR but please could you update these workspace versions to the latest published ones in your next PR?

**Review by kevinrodrigues — DISMISSED** [review]

**Review by xander-marjoram — DISMISSED** [review]

**Review by ashleynolan — APPROVED** [review]

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -35,6 +35,7 @@ env:
   PERCY_TOKEN_PIE_NOTIFICATION: ${{ secrets.PERCY_TOKEN_PIE_NOTIFICATION }}
   PERCY_TOKEN_PIE_SWITCH: ${{ secrets.PERCY_TOKEN_PIE_SWITCH }}
   PERCY_TOKEN_PIE_SPINNER: ${{ secrets.PERCY_TOKEN_PIE_SPINNER }}
+  PERCY_TOKEN_PIE_COMPONENT_NAME: ${{ secrets.PERCY_TOKEN_PIE_TAG }}
```
Comment (siggerzz) [inline .github/workflows/ci.yml:38]: This should probably be:

`PERCY_TOKEN_PIE_TAG: ${{ secrets.PERCY_TOKEN_PIE_TAG }}`

**Review by siggerzz — CHANGES_REQUESTED** [review]
Small comment!

File: .github/workflows/ci.yml (hunk 2 above)
Comment (dandel10n) [reply .github/workflows/ci.yml:38]: Oops

File: packages/components/pie-tag/package.json (hunk 1 above)
Comment (dandel10n) [reply packages/components/pie-tag/package.json]: Since I needed to change token name I added your suggestion as well 👍

**Review by siggerzz — APPROVED** [review]

**Review by xander-marjoram — APPROVED** [review]

**Pull request merged**